zerver/models.py

```diff
@@ -696,6 +712,11 @@ def get_giphy_rating_options(self) -> Dict[str, Dict[str, object]]:
     # they will not be available regardless of users' personal settings.
     enable_read_receipts = models.BooleanField(default=False)
 
+    group_property_mapping: Dict[str, str] = dict(
+        direct_message_initiator_group="direct_message_initiator_group_id",
+        direct_message_permission_group="direct_message_permission_group_id",
+    )
```

Comment:
This dict feels a little silly, if it's always going to be just appending `_id`.